hooks/purge.ps1

```diff
@@ -13,7 +13,12 @@ param(
 
 pslist -t -accepteula -nobanner
 
-$absDir = Resolve-Path $Dir
+if (Test-Path -Path $Dir) {
+	$absDir = Resolve-Path $Dir -ErrorAction Stop
+} else {
+	"Directory $Dir does not exist, skipping"
```

Comment:
```suggestion
	Write-Output "Skipping: Directory $($Dir) does not exist, no file-handles can exist."
```

Reply:
Accepted + updated to use `Write-Output` throughout